features/step_definitions/manual_steps.rb

```diff
@@ -278,9 +278,9 @@
 When(/^I add another section and publish the manual later$/) do
   create_section(
     @manual.title,
-    section_title: "Another section so we can publish",
-    section_summary: "Another section so we can publish summary",
-    section_body: "Another section so we can publish body",
+    { section_title: "Another section so we can publish",
+      section_summary: "Another section so we can publish summary",
+      section_body: "Another section so we can publish body" },
```

Comment:
It's not clear to me why this was changed? does it fix a problem? if so it probably should be a separate commit.

Reply:
It's necessary to fix error from refactoring to support manuals + sections work - a hint to ruby parser 

I believe the error was caused to adding an optional "save" parameter to create_section, along the lines of existing create_manual method, which means these existing parameters need to be passed as an explicit hash not gathered into a hash by default ruby parsing.

Other call sites don't get caught by this error as they reference the hash of fields by variable rather than automatic gathering in this one call.

Reply:
Ah yes, that makes sense, now you've introduced a keyword argument on the method it needs this to be distinct from them. It'd be useful having the information you've provided there with the commit.